<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaffee Automat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="CoffeeMachine" name="coffeeMachine" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="traficAnalysisDockWidget">
   <property name="windowTitle">
    <string>Signal Tracing</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="TraficAnalysis" name="traficAnalysis"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionSignalTracing"/>
   <addaction name="separator"/>
   <addaction name="actionCheckStatemachine"/>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSignalTracing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Signal Tracing</string>
   </property>
  </action>
  <action name="actionCheckStatemachine">
   <property name="text">
    <string>Check Statemachine</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CoffeeMachine</class>
   <extends>QWidget</extends>
   <header>CoffeeMachine.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TraficAnalysis</class>
   <extends>QWidget</extends>
   <header>TraficAnalysis.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>reset_pressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSignalTracing</sender>
   <signal>triggered(bool)</signal>
   <receiver>traficAnalysisDockWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>traficAnalysisDockWidget</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionSignalTracing</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCheckStatemachine</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>check_statemachine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reset_pressed()</slot>
  <slot>check_statemachine()</slot>
 </slots>
</ui>
